UI/index.html

```diff
@@ -0,0 +1,221 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/html">
+<head>
+    <meta charset="UTF-8">
+    <title>Sweet Tweet</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+<body>
+<header>
+    <div id="header">  <h1> Sweet Tweet </h1> </div>
+    <p class="user"> Alexander T </p>
+    <div id="border"> </div>
+</header>
+<main>
+    <div id="news-line">
+        <div class="post">
+            <div class="post-header">
+                <div class="username">
+                    Alexander T
+                </div>
+                <div class="user-buttons">
+                    <form>
+                        <button class="user-button">Edit Post</button>
+                        <button class="user-button"> Delete Post </button>
+                    </form>
+                </div>
+            </div>
+            <p>
+                America’s music culture would be incomplete without
+                blues music. Thought it was created in the early decades of the 20th century,
+                blues music has had a huge influence on American popular music
+                up to the present days.
+            </p>
+            <br><h>#music #america #bluise</h><br/>
+            <div class="post-footer">
+                <div class="right">
```

Comment:
I can't find any styles in css for that class. Also, it is called `right`, but button is located at the left side of post